src/lib/core/components/filter/HistogramFilter.tsx

```diff
@@ -570,6 +570,8 @@ function HistogramPlotWithControls({
         selectedRange={selectedRange}
         selectedRangeBounds={selectedRangeBounds}
         onSelectedRangeChange={handleSelectedRangeChange}
+        //DKDK parentComponentName
+        parentComponentName={'HistogramFilter'}
```

Comment:
Hi @moontrip  - I was going to give this PR a quick review, but I noticed it doesn't compile because the `parentComponentName` props are still here.  Does that mean it's not ready for review?  I know this is second-priority stuff due to the b56 deadline but I thought yesterday you said it was done.

Reply:
Oh, I see.  You're now working in the branch `pr794-suggestion`

I thought you would merge my suggestions into your branch, and continue there.

But it's OK, I have it working now!